src/main/kotlin/gg/essential/elementaPractice/gui/WhiteboardGUI.kt

```diff
@@ -0,0 +1,205 @@
+package gg.essential.elementaPractice.gui
+
+import gg.essential.elementa.ElementaVersion
+import gg.essential.elementa.WindowScreen
+import gg.essential.elementa.components.UIBlock
+import gg.essential.elementa.components.UIRoundedRectangle
+import gg.essential.elementa.components.UIText
+import gg.essential.elementa.constraints.CenterConstraint
+import gg.essential.elementa.constraints.ChildBasedMaxSizeConstraint
+import gg.essential.elementa.constraints.ImageAspectConstraint
+import gg.essential.elementa.constraints.animation.Animations
+import gg.essential.elementa.dsl.*
+import gg.essential.elementa.effects.OutlineEffect
+import gg.essential.elementaPractice.events.WhiteboardEvent
+import gg.essential.elementaPractice.gui.components.ColorToolComponent
+import gg.essential.elementaPractice.gui.components.SliderComponent
+import gg.essential.elementaPractice.gui.components.WhiteboardComponent
+import gg.essential.universal.UMinecraft
+import net.minecraftforge.common.MinecraftForge
+import java.awt.Color
+
+public class WhiteboardGUI : WindowScreen(ElementaVersion.V1) {
+
+    private val whiteboardEvent = WhiteboardEvent(this)
+    private var open = false
+
+    init {
+
+        // TO-DO:
+        // Save/Load buttons?
+
+        // Button to close the whiteboard
+        val closeButton = UIBlock(Color(207, 207, 196)).constrain {
```

Comment:
When working on Essential UIs, we use a class to store all the colors for UIs so that if we want to change a color, we only need to change it in one place instead of every place it's used. Check out the VigilancePalette and EssentialPalette classes.